### ISSUE TITLE
git-upload-pack progress

### ISSUE THREAD

**Issue by TranceGeniK** [isomorphic-git/isomorphic-git#1243]
Is there a way to get the progress while the git-upload-pack is downloaded ?
It seems that all the events sent by onProgress are on pause when downloading the data, it would be great to see the progress of the data being downloading

**Comment by jcubic** [comment]
I don't think it will be possible anytime soon. But I'm happy to merge a PR that add this feature.
